### ISSUE TITLE
[Android] ToolbarAndroid Logo and Action Images are small

### ISSUE THREAD

**Issue by malinthe** [facebook/react-native#4041]
After switching to the new asset system on RN 0.14.2, the images loaded by ToolbarAndroid are extremely small. When loading the images with ```<Image>```, there doesn't seem to be any issue. 

Packager showed that it loaded a @3x.png - a 72x72 image (a 24dp material icon from drawable-xxhdpi)

Have I got the sizing of images wrong?

Toolbar View:

```javascript
var HeaderView = React.createClass({
    actions: function () {
        return [
            {title: 'Search', icon: require('./img/search.png'), show: 'always'},
            {title: 'Reload', icon: require('./img/reload.png'), show: 'always'},
            {title: 'About'}
        ];
    },

    render: function () {
        return (
            <ToolbarAndroid logo={require('./img/logo.png')} titleColor="#FFFFFF"
             title="Title" style={styles.toolbar} actions={this.actions()} 
             onActionSelected={this.selectAction}/>
        )
    }
});
```

This is the view I'm getting:

![screenshot_2015-11-10-16-57-00_1](https://cloud.githubusercontent.com/assets/1997002/11061455/4f357650-87cd-11e5-8f0b-0fe971931953.jpg)

**Comment by cosmith** [comment]
I have the same issue, it was working fine with `image!...`

**Comment by BigPun86** [comment]
Me too! I solved this by generating me a new image file from  [Android Assets](https://romannurik.github.io/AndroidAssetStudio/index.html)

After selecting Action bar and tab icons it generates the image i need...But i am still not sure why this was changed. It´s nice to have a shared image Folder for both platforms, but it caused me a lot of other problems. When i build my apk and install it to my device, it doesn´t show me any images....

**Comment by marty-wang** [comment]
Same issue for me.

**Comment by marsprince** [comment]
me,too

**Comment by cosmith** [comment]
Generating images with the Android Assets icon generator worked. I think there was a bug before in the sizing of the images, and that the new asset system fixed it.

**Comment by malinthe** [comment]
@BigPun86 @cosmith Could you explain how you generated the images with Asset Icon Generator? I tried generating Action Bar and Tab Icons but the sizes were unchanged.

**Comment by cosmith** [comment]
You have to take each generated image in its own folder and rename it to the corresponding size:

before | after
-----------------|-------------------
mdpi/image.png | image.png
hdpi/image.png | image@1.5x.png
xhdpi/image.png | image@2x.png
xxhdpi/image.png | image@3x.png
xxxhdpi/image.png | image@4x.png

**Comment by malinthe** [comment]
@cosmith I renamed exactly as you mentioned but they're still small. I'm running the app on a 1080x1920 device. Could that be the issue?

**Comment by cosmith** [comment]
I'm using a Nexus 5 which is also 1080p. 

Try bundling the assets like this:
```
react-native bundle --platform android --dev true --entry-file index.android.js   --bundle-output android/app/src/main/assets/index.android.bundle   --assets-dest android/app/src/main/res/
```

**Comment by marty-wang** [comment]
I used sketchapp to generate images of different sizes. The images seem fine and correct. What would be the difference from the images generated from Android asset icon generator? As long as the image size is correct, why does it matter where/how it is generated?

**Comment by malinthe** [comment]
@cosmith Still no change. Would it possible for you to upload one set of images you're using so I could check if the sizes are accurate?

**Comment by malinthe** [comment]
@cosmith Success! Bundling worked. I had trouble with the command on Windows for some reason. Tried it on the Linux box and everything seems to be okay now.

**Comment by malinthe** [comment]
@cosmith So on initial load the images show proper sizes but if I reload the JS, the images go back to being small. Packager misbehaving?

**Comment by doronnimrod** [comment]
Im having a different issue 
when i set logo  `logo={require('./images/logo-trans.png')}`
or action icon i get the following error , RN 0.14.2
 `TypeError:expected dynamic type 'string' but had type 'object'`

anybody ?

**Comment by richarddewit** [comment]
+1

**Comment by cosmith** [comment]
Yeah this is actually still broken for me too, but only if I load the bundle dynamically and not if I build it and load from disk, so it's not an issue in production.

**Comment by malinthe** [comment]
@cosmith You're right. I tried building a release version and it had no issues.

**Comment by richarddewit** [comment]
Using `0.16.0-rc` fixes it

**Comment by malinthe** [comment]
Seems to be fixed in 0.16
